newZealand.html

```diff
@@ -25,7 +24,7 @@ <h1>New Zealand</h1>
             <div class="agency">
                 <h2>Get Lost Travel Agency</h2>
             </div>
-            <img class="logo vish" src="./images/travel_logo.png" alt="Travel Logo">
+            <img class="logo vish" src="./images/newzealand_logo.png" alt="newzealand_logo">
         </header>
     </div>
     <main class="kiwiContainer">
```

Comment:
needs <ul> on next line, to complete the "unordered list" thought, before continuing.